### PR TITLE
Updating the Application compose file

### PR DESCRIPTION
**Pull request by devopseasylearning**
Updating the Application compose file

### CODE REVIEW

File: compose.yml [hunk 1]
```diff
@@ -11,11 +11,7 @@ services:
     ports:
       - "8002:80"
```
Comment (devopseasylearning) [inline compose.yml:13]: why are you deleting the block below

Comment (devopseasylearning) [reply compose.yml:13]: @s7rosine what is your take on this

File: compose.yml [hunk 2]
```diff
@@ -29,13 +25,6 @@ services:
     ports:
```
Comment (s7rosine) [inline compose.yml:25]: @s7maryam what is your decision now?

File: compose.yml (hunk 1 above)
Comment (olamyde) [reply compose.yml:13]: Good to go

File: compose.yml [hunk 3]
```diff
@@ -11,11 +11,7 @@ services:
...
       - "8002:80"
 
-  nginx3:
-    image: nginx:latest
-    ports:
-      - "8003:80"
-  nginx3:
+
```
Comment (NtBahati) [inline compose.yml:14]: I am not sure what you wanted to do here.

File: compose.yml (hunk 1 above)
Comment (mandengpierre) [reply compose.yml:13]: The Project is sweet

**Comment by versage1** [comment]
YES IT IS CONFIRMED  GUYS

**Review by 9A4119 — COMMENTED** [review]
Can we get a better image please?

File: s7-file2 [hunk 4]
```diff
@@ -11,3 +11,10 @@ spec:
     metadata:
       labels:
         app: sample-app
```
Comment (Agbor95) [inline s7-file2:13]: @s7eric why did you add this line

**Comment by S7belliss** [comment]
@ good man !

File: compose.yml [hunk 5]
```diff
@@ -29,13 +25,6 @@ services:
     ports:
       - "8005:80"
   nginx4:
-    image: nginx:latest
-    ports:
-      - "8004:80"
-
-  nginx5:
-    image: nginx:latest
```
Comment (s7rosine) [inline compose.yml:37]: @s7solomon  i see you made a very good Dockerfile

File: compose.yml (hunk 1 above)
Comment (devopseasylearning) [reply compose.yml:13]: hummm I dont  think so

**Comment by EDEMOS80** [comment]
i see your modification and i think it is okay

File: compose.yml (hunk 1 above)
Comment (nalukunyang) [reply compose.yml:13]: @devopseasylearning, i think this was discussed during previous meeting

File: s7-file2 [hunk 6]
```diff
@@ -11,3 +11,10 @@ spec:
...
+  selector:
+    matchLabels:
+      app: sample-app
+  template:
```
Comment (s7jackye) [inline s7-file2:17]: What is this part showing?

File: compose.yml [hunk 7]
```diff
@@ -11,11 +11,7 @@ services:
     ports:
       - "8002:80"
 
-  nginx3:
-    image: nginx:latest
-    ports:
-      - "8003:80"
```
Comment (9A4119) [inline compose.yml:17]: Did you bind this port?

**Comment by CHOUPAGUY** [comment]
Can you  review your changes

File: compose.yml (hunk 1 above)
Comment (Agbor95) [reply compose.yml:13]: i like this

File: compose.yml (hunk 3 above)
Comment (Ajmensah) [reply compose.yml:14]: @NtBahati good

Comment (s7rosine) [reply compose.yml:14]: @NtBahati i was trying to make sure i am using the right base image

File: compose.yml (hunk 1 above)
Comment (Djurizt) [reply compose.yml:13]: block of code

File: compose.yml (hunk 2 above)
Comment (9A4119) [reply compose.yml:25]: Looks good to me.

File: compose.yml (hunk 1 above)
Comment (RasheedatA) [reply compose.yml:13]: Agreed

Comment (THIMONE-NOUPA) [reply compose.yml:13]: ok. my comment is already there

File: compose.yml [hunk 8]
```diff
@@ -11,11 +11,7 @@ services:
     ports:
       - "8002:80"
 
-  nginx3:
-    image: nginx:latest
```
Comment (sadioci2) [inline compose.yml:15]: what are willing to do here?

File: s7-file2 (hunk 4 above)
Comment (EDEMOS80) [reply s7-file2:13]: i am also wondering about that add

File: compose.yml [hunk 9]
```diff
@@ -29,13 +25,6 @@ services:
     ports:
       - "8005:80"
   nginx4:
-    image: nginx:latest
-    ports:
-      - "8004:80"
-
-  nginx5:
```
Comment (sadioci2) [inline compose.yml:36]: shouldn't be using the latest version

File: compose.yml (hunk 1 above)
Comment (Perrine53) [reply compose.yml:13]: Good job!

Comment (THIMONE-NOUPA) [reply compose.yml:13]: finally!
 i was able to comment....

**Review by sadioci2 — COMMENTED** [review]
please check my comments

**Comment by Agbor95** [comment]
Good work guys

File: compose.yml (hunk 5 above)
Comment (EDEMOS80) [reply compose.yml:37]: but need a little modification

File: compose.yml (hunk 3 above)
Comment (THIMONE-NOUPA) [reply compose.yml:14]: Me too.

**Comment by MNgouhouo** [comment]
I like this collaboration

**Comment by mandengpierre** [comment]
sweet class.

**Comment by versage1** [comment]
good

**Comment by s7eric** [comment]
yes

File: compose.yml (hunk 1 above)
Comment (Sekoutou09) [reply compose.yml:13]: Ok to merge

File: compose.yml (hunk 7 above)
Comment (Sekoutou09) [reply compose.yml:17]: Same question applied 🥺

File: s7-file2 (hunk 6 above)
Comment (Sekoutou09) [reply s7-file2:17]: Good to merge ! 👍

**Review by NtBahati — COMMENTED** [review]
Looks good

**Comment by NtBahati** [comment]
@Sekoutou09 Thank you for asking me to review your code. All looks ok.

**Review by versage1 — COMMENTED** [review]
sounds good

**Comment by sandargo** [comment]
The file has been well refined and looks good now.
